index.js

```diff
@@ -11,5 +13,5 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
 app.listen(PORT, () => {
-  console.log(`connected on port ${PORT}`);
-});
\ No newline at end of file
+  nodeLogger.info(`connected on port ${PORT}`);
```

Comment:
Use `logger` instead of `nodeLogger` which is too verbose

Reply:
Thank you for the feed back i would rename to logger variable